Watch Extension/State Management/ContextReducer.swift

```diff
@@ -4,18 +4,27 @@ import WatchConnectivity
 
 enum ContextAction {
     case requestFullContext
+    case fullContextRequestFailed
 }
 
 // TODO: need to figure out how to remove the state from here since
 // there is no need to have it at all. This reducer is to just formalize
 // an action
-func contextReducer(state _: inout AppState, action: ContextAction) -> [Effect<ContextAction>] {
+func contextReducer(showCommunicationErrorFyiDialog: inout Bool, action: ContextAction) -> [Effect<ContextAction>] {
     switch action {
     case .requestFullContext:
-        let communication = WatchContextCommunication()
-        let encoder = JSONEncoder()
-        guard let data = try? encoder.encode(communication) else { return [] }
-        WCSession.default.sendMessageData(data, replyHandler: nil, errorHandler: nil)
+        return [{ callback in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/69b030c9a02790233ca920809da0d6dc623dd3f8/Watch Extension/State Management/ContextReducer.swift#L16"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>ContextReducer.swift:16</code></td>
    </tr>
  </tbody>
</table>